content/applications/hr/payroll.rst

```diff
@@ -286,21 +426,15 @@ When offering potential employees a position, there can be certain *advantages*
 addition to the salary to make the offer more appealing (such as extra time off, access to a company
 car, reimbursement for a phone or internet, etc.).
 
-To see the advantages, go to :menuselection:`Payroll --> Configuration --> Advantages`. Advantages
-are grouped by :guilabel:`Structure type`.
+To see the advantages, go to :menuselection:`Payroll --> Configuration --> Salary Package
+Configurator --> Advantages`. Advantages are grouped by :guilabel:`Structure type`, and the
+advantage listed for a particular structure type is only available for that specific structure.
```

Comment:
This last sentence needs explanation it sounds confusing. Maybe an example block

Reply:
Added an example! Might be verbose, but it makes this clear.